app/sidebar.py

```diff
@@ -7,7 +7,15 @@
 from ptxboa.api import PtxboaAPI
 
 
+@st.cache_resource()
+def sidebar_logo():
+    st.image("img/agora-energiewende_logo_612x306.png")
+
+
 def make_sidebar(api: PtxboaAPI):
+    with st.sidebar:
+        sidebar_logo()
+

```

Comment:
I tried the new st.logo. You can uncomment this to see how it looks with texts removed from the logos.